docs/source/releases/5.2.x.rst

```diff
@@ -2,6 +2,18 @@
 Django MongoDB Backend 5.2.x
 ============================
 
+5.2.1
+=====
+
+*Unreleased*
+
+New features
+------------
+
+- Allowed specifying the MongoDB connection string in ``DATABASES["HOST"]``,
+  eliminating the need to use :func:`~django_mongodb_backend.utils.parse_uri`
```

Comment:
Until we are sure, I'd probably refrain from declaring that we've eliminated the need for `parse_uri`. For example, can this be described as implementing a similar feature set as `dj-database-url` and if so, is this approach something that could eliminate the need for that 3rd party library in Django (via some upstream changes to Django) ?

Reply:
> This is somewhat orthogonal to dj-database-url

Yes it is but `parse_uri` is not (orthogonal to `dj-database-url`). So this does not eliminate the need for `parse_uri`.

Reply:
I'm not sure what you mean. Are you suggesting not to deprecate parse_uri? What use case for `parse_uri` do you have in mind?

Reply:
After more consideration, it seems feasible to support using the connection string's "defaultauthdb" as the database so that NAME isn't required. It's consistent with the fact that `MongoClient.get_database()` returns that db. The name does need to be parsed out because Django relies on `settings_dict["NAME"]` in [django/db/backends/base/creation.py](https://github.com/django/django/blob/main/django/db/backends/base/creation.py).

Reply:
Support for /dbname is added in commit "parse db name out of connection string".